custom_components/mail_and_packages/const.py

```diff
@@ -1099,4 +1137,5 @@
     "purolator",
     "intelcom",
     "post_nl",
+    "ebay",
```

Comment:
```suggestion
```
Since eBay doesn't actually deliver packages we wouldn't add a "delivering" sensor.